manifests/v3.10.0/apiservice-cabundle-controller/deployment.yaml

```diff
@@ -0,0 +1,53 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  namespace: openshift-service-cert-signer
+  name: apiservice-cabundle-injector
+  labels:
+    app: openshift-apiservice-cabundle-injector
+    apiservice-cabundle-injector: "true"
+spec:
+  strategy:
+    type: Recreate
+  selector:
+    matchLabels:
+      app: openshift-apiservice-cabundle-injector
+      apiservice-cabundle-injector: "true"
+  template:
+    metadata:
+      name: apiservice-cabundle-injector
+      labels:
+        app: openshift-apiservice-cabundle-injector
+        apiservice-cabundle-injector: "true"
+    spec:
+      serviceAccountName: apiservice-cabundle-injector-sa
+      containers:
+      - name: apiservice-cabundle-injector-controller
+        image: ${IMAGE}
+        imagePullPolicy: IfNotPresent
```

Comment:
Do we want to make this configurable? I guess in production you want to have this Always?

Reply:
> Do we want to make this configurable? I guess in production you want to have this Always?

It is configurable via the operator CR.